packages/web/src/scss/components/Modal/_Modal.scss

```diff
@@ -0,0 +1,93 @@
+@use 'sass:map';
+@use '../../tools/accessibility';
+@use '../../tools/breakpoint';
+@use '../../tools/svg';
+@use 'theme';
+
+.Modal {
+    position: fixed;
+    top: 0;
+    overflow-x: hidden;
+    overflow-y: auto;
```

Comment:
To my surprise, it seems `-webkit-overflow-scrolling: touch` is not necessary any longer…

> Added support for one-finger accelerated scrolling to all frames and `overflow:scroll` elements eliminating the need to set `-webkit-overflow-scrolling: touch`.

https://developer.apple.com/documentation/safari-release-notes/safari-13-release-notes